posix.ipkg

```diff
@@ -3,6 +3,14 @@ package posix
 sourcedir = src
 modules = System.Posix.Directory
         , System.Posix.Time
+	, System.Posix.Signal
+	, System.Posix.Syslog
+	, System.Posix.Test.Signal
+	, System.Posix.Test.Syslog
```

Comment:
Okay, well, in my browser it appears the tabs are displaying equal to seven spaces for some reason, so it caught my eye immediately. And in my mail client they appear equal to five spaces. I have no idea why.